processor/groupbytraceprocessor/config.go

```diff
@@ -0,0 +1,46 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package groupbytraceprocessor
+
+import (
+	"time"
+
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config is the configuration for the processor.
+type Config struct {
+	configmodels.ProcessorSettings `mapstructure:",squash"`
+
+	// NumTraces is the max number of traces to keep in memory waiting for the duration.
+	// Default: 1.000.000.
```

Comment:
nit: `1,000,000` format is more common in the English speaking world, I believe.
https://en.wikipedia.org/wiki/Decimal_separator#Examples_of_use

Reply:
Sure, I can change, but is it ambiguous the way it is?

Reply:
1_000_000 seems fine.